actionpack/test/controller/rescue_test.rb

```diff
@@ -306,6 +314,16 @@ def test_block_rescue_handler_with_argument_as_string
     get :exception_with_no_handler_for_wrapper
     assert_response :unprocessable_entity
   end
+
+  test "can rescue a ParserError" do
+    silence_logger
+    post :arbitrary_action, body: "{", as: :json
+    assert_response :success
```

Comment:
`assert_response :ok` 

Could we make an assertion on the body's content to ensure that the response came from rescuing the exception